src/My.Extensions.Localization.Json/JsonMvcDataAnnotationsMvcBuilderExtensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+
+using Microsoft.AspNetCore.Mvc.DataAnnotations;
+using Microsoft.Extensions.Localization;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public static class JsonLocalizationMvcDataAnnotationsMvcBuilderExtensions
```

Comment:
The name is ugly little bit can we suggest a better one

Reply:
Yes I did not like it but tried to keep the convention :)
What about `JsonLocalizationMvcDataAnnotationsExtensions` which is still long; But it is a little bit shorter?